mteb/tasks/Classification/multilingual/IndicNLPNewsClassification.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks import AbsTaskClassification, MultilingualTask
+
+_LANGUAGES = {
+    "gu": ["guj-Gujr"],
+    "kn": ["kan-Knda"],
+    "mal": ["mal-Mlym"],
+    "mr": ["mar-Deva"],
+    "tel": ["tel-Telu"],
+    "ori": ["ori-Orya"],
+    "pa": ["pan-Guru"],
+    "ta": ["tam-Taml"],
+}
+
+
+class IndicNLPNewsClassification(MultilingualTask, AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="IndicNLPNewsClassification",
+        dataset={
+            "path": "Sakshamrzt/IndicNLP-Multilingual",
+            "revision": "308230fabdcada106869a5196ba11c403d065b07",
+        },
+        description="A News classification dataset in multiple Indian regional languages.",
+        reference="https://github.com/AI4Bharat/indicnlp_corpus/blob/master/ai4bharat-indicnlp-corpus-2020.pdf",
```

Comment:
We already have this dataset for Indian languages: [IndicSentimentClassification](https://github.com/embeddings-benchmark/mteb/blob/main/mteb/tasks/Classification/multilingual/IndicSentimentClassification.py). Is the dataset you're adding different from this one?

Reply:
Okay, can you please add more information about the classes in the dataset card?

Reply:
Added